apps/hash-ai-worker-py/src/worker/tests/mocks/activity.py

```diff
@@ -69,7 +68,46 @@ async def graph_workflow(
     }
 
 
+async def delegate_aipy_queue(
+    activity: str,
+    arg: Any,  # noqa: ANN401
+    *,
+    start_to_close_timeout: timedelta,  # noqa: ARG001
+) -> Any:  # noqa: ANN401
+    match activity:
+        case "inferEntities":
+            return (await infer_entities(arg)).model_dump(by_alias=True)
+        case _:
+            msg = f"Unknown activity: `{activity}`"
+            raise ValueError(msg)
+
+
+async def graph_workflow(
+    activity: str,
+    arg: Any,  # noqa: ANN401
+    *,
+    task_queue: str = "aipy",
```

Comment:
would it make sense to restrict this further to Literal?

Reply:
No, because this is not a Literal but a variable.